### PR TITLE
Cleanup: Remove some rowwise loading code

### PR DESCRIPTION
**Pull request by sduzh**

### CODE REVIEW

**Review by wyb — DISMISSED** [review]

File: be/src/exec/tablet_sink.cpp [hunk 1]
```diff
@@ -137,7 +130,7 @@ void NodeChannel::open() {
         request.set_load_mem_limit(_parent->_load_mem_limit);
     }
     request.set_load_channel_timeout_s(_parent->_load_channel_timeout_s);
-    request.set_is_vectorized(_parent->_is_vectorized);
+    request.set_is_vectorized(true);
```
Comment (chaoyli) [inline be/src/exec/tablet_sink.cpp:133]: Could you set the is_vectorized deprecated in thrift?

Comment (sduzh) [reply be/src/exec/tablet_sink.cpp:133]: OK

Comment (sduzh) [reply be/src/exec/tablet_sink.cpp]: updated, PTAL @chaoyli @wyb

File: gensrc/proto/internal_service.proto [hunk 2]
```diff
@@ -133,7 +133,7 @@ message PTabletWriterOpenRequest {
     required bool need_gen_rollup = 7; // Deprecated
     optional int64 load_mem_limit = 8;
     optional int64 load_channel_timeout_s = 9;
-    optional bool is_vectorized = 20;
+    optional bool DEPRECATED_is_vectorized = 20;
```
Comment (chaoyli) [inline gensrc/proto/internal_service.proto:136]: You can not change the variable name. Instead, add a DEPRECATED flag.
```
optional bool is_vectorized = 20; // DEPRECATED
```

**Review by chaoyli — CHANGES_REQUESTED** [review]

File: gensrc/proto/internal_service.proto (hunk 2 above)
Comment (sduzh) [reply gensrc/proto/internal_service.proto:136]: This variable is no longer used, IMHO, renaming it would be more clear.

Comment (chaoyli) [reply gensrc/proto/internal_service.proto:136]: Change variable is a dangerous thing. The standard method are add an annotation, to indicate it.

Comment (sduzh) [reply gensrc/proto/internal_service.proto:136]: Did you mean the `deprecated` field option in protobuf?  It's written as `[deprecated = true]`, what you wrote is a c++ comment.

Comment (chaoyli) [reply gensrc/proto/internal_service.proto]: you can use [deprecated = true]

**Review by chaoyli — DISMISSED** [review]

**Review by imay — APPROVED** [review]

File: gensrc/proto/internal_service.proto (hunk 2 above)
Comment (sduzh) [reply gensrc/proto/internal_service.proto]: done

**Comment by chaoyli** [comment]
run starrocks_be_unittest

**Review by chaoyli — APPROVED** [review]

**Pull request merged**